.github/workflows/ci.yml

```diff
@@ -11,3 +11,12 @@ jobs:
 
       - name: js-unit-tests
         run: yarn test
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build
+        uses: loxq/jekyll-build-action@master
```

Comment:
Can we try to find a jekyll build action that's more supported/standard?

https://github.com/loxq/jekyll-build-action Using forks will not suffice as a long=term solution.

Reply:
Agree. I did not find any GitHub Actions that does just the build. What about using the [builder](https://hub.docker.com/r/jekyll/builder) docker image directly?

Reply:
So if you look in the root directory we already have a docker-compose.yml file. This is what I've been using for local dev, because setting up ruby/jekyll locally is kind of a pain. What about we try to use that instead?

Reply:
I thought of adding something like this in the `ci.yml` file:
```yaml
  build:
    runs-on: ubuntu-latest
    container: jekyll/builder:4.0.1
    steps:
      - uses: actions/checkout@v2
      - name: jekyll-build
        run: |
          mkdir _site .jekyll-cache
          jekyll build --trace
```
or do you prefer to put this in the `docker-compose.yml` file?